kinder/ci/scripts/kinder-run.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Starting the kinder workflow runner..."
+
+# this script should be used as a bridge between CI jobs and the kinder binary
+if [ "$#" -eq 0 ]; then
+    echo "error: missing argument for workflow config"
+    echo "usage:"
+    echo "# to pick a workflow .yaml file by name from /ci/workflows/"
+    echo "kinder-run.sh some-workflow"
+    echo "# to use a file from an absolute path"
+    echo "kinder-run.sh /some-path/some-workflow.yaml"
+    exit 1
+fi
+
+# set paths
+CURRENT_PATH="$(pwd)"
+SCRIPT_PATH="$(dirname "$(readlink -f "$0")")"
+ROOT_PATH="$(readlink -f "${SCRIPT_PATH}/../..")"
```

Comment:
What about changing "${SCRIPT_PATH}/../..")" and "${ROOT_PATH}/ci/workflows" into "${SCRIPT_PATH}/..")" and "${ROOT_PATH}/workflows", so we are removing the assumption about the containing folder named ci?
What about moving kinder-run.sh into ci folder, so we are removing the assumption of having a sibling folder named workflows (but we are still assuming a workflows child folder)